### DISCUSSION TITLE
Properly submitting images to frigate+

### DISCUSSION THREAD

**Discussion by ibrewster** [blakeblackshear/frigate#14176] [Ask A Question] [answered]
I'm trying to figure out what the proper way to submit some of my images to frigate+ is. Take the image attached here. frigate properly identified a car, but was wildly inaccurate as to which portion of the image was a car. So do I submit this as a true positive, and fix the bounding box, or submit it as a false positive, and mark the correct bounding box for the car?

I've been submitting these as true positives, since it *is* a car, but I realized I might have been doing it wrong...

<img width="2156" alt="Screenshot 2024-10-06 at 7 13 16 AM" src="https://github.com/user-attachments/assets/59d612ad-fd42-485c-a42f-bdfd66b91f29">

**Comment by NickM-27** [comment, marked as answer]
You say `yes this is a car` and then fix the box to be correct
